diff --git a/node_modules/@coinflowlabs/react/build/cjs/common/CoinflowLibMessageHandlers.js b/node_modules/@coinflowlabs/react/build/cjs/common/CoinflowLibMessageHandlers.js
index 0abe0b7..01d0f50 100644
--- a/node_modules/@coinflowlabs/react/build/cjs/common/CoinflowLibMessageHandlers.js
+++ b/node_modules/@coinflowlabs/react/build/cjs/common/CoinflowLibMessageHandlers.js
@@ -133,6 +133,11 @@ function getHandlers(props) {
             onSuccess: props.onSuccess,
             onAuthDeclined: props.onAuthDeclined,
         }),
+        tempo: () => getEvmWalletHandlers({
+            wallet: wallet,
+            onSuccess: props.onSuccess,
+            onAuthDeclined: props.onAuthDeclined,
+        }),
         arbitrum: () => getEvmWalletHandlers({
             wallet: wallet,
             onSuccess: props.onSuccess,
diff --git a/node_modules/@coinflowlabs/react/build/cjs/common/CoinflowUtils.js b/node_modules/@coinflowlabs/react/build/cjs/common/CoinflowUtils.js
index a18e5a6..3fb26b2 100644
--- a/node_modules/@coinflowlabs/react/build/cjs/common/CoinflowUtils.js
+++ b/node_modules/@coinflowlabs/react/build/cjs/common/CoinflowUtils.js
@@ -217,6 +217,12 @@ class CoinflowUtils {
                 const { transaction } = props;
                 return lz_string_1.default.compressToEncodedURIComponent(JSON.stringify(transaction));
             },
+            tempo: () => {
+                if (!('transaction' in props))
+                    return undefined;
+                const { transaction } = props;
+                return lz_string_1.default.compressToEncodedURIComponent(JSON.stringify(transaction));
+            },
             arbitrum: () => {
                 if (!('transaction' in props))
                     return undefined;
@@ -253,6 +259,8 @@ class CoinflowUtils {
                 return args.eth;
             case 'base':
                 return args.base;
+            case 'tempo':
+                return args.tempo;
             case 'arbitrum':
                 return args.arbitrum;
             case 'stellar':
diff --git a/node_modules/@coinflowlabs/react/build/esm/common/CoinflowLibMessageHandlers.js b/node_modules/@coinflowlabs/react/build/esm/common/CoinflowLibMessageHandlers.js
index 7935dc7..e60c876 100644
--- a/node_modules/@coinflowlabs/react/build/esm/common/CoinflowLibMessageHandlers.js
+++ b/node_modules/@coinflowlabs/react/build/esm/common/CoinflowLibMessageHandlers.js
@@ -127,6 +127,11 @@ export function getHandlers(props) {
             onSuccess: props.onSuccess,
             onAuthDeclined: props.onAuthDeclined,
         }),
+        tempo: () => getEvmWalletHandlers({
+            wallet: wallet,
+            onSuccess: props.onSuccess,
+            onAuthDeclined: props.onAuthDeclined,
+        }),
         arbitrum: () => getEvmWalletHandlers({
             wallet: wallet,
             onSuccess: props.onSuccess,
diff --git a/node_modules/@coinflowlabs/react/build/esm/common/CoinflowUtils.js b/node_modules/@coinflowlabs/react/build/esm/common/CoinflowUtils.js
index 48bf27d..b1567e3 100644
--- a/node_modules/@coinflowlabs/react/build/esm/common/CoinflowUtils.js
+++ b/node_modules/@coinflowlabs/react/build/esm/common/CoinflowUtils.js
@@ -212,6 +212,12 @@ export class CoinflowUtils {
                 const { transaction } = props;
                 return LZString.compressToEncodedURIComponent(JSON.stringify(transaction));
             },
+            tempo: () => {
+                if (!('transaction' in props))
+                    return undefined;
+                const { transaction } = props;
+                return LZString.compressToEncodedURIComponent(JSON.stringify(transaction));
+            },
             arbitrum: () => {
                 if (!('transaction' in props))
                     return undefined;
@@ -248,6 +254,8 @@ export class CoinflowUtils {
                 return args.eth;
             case 'base':
                 return args.base;
+            case 'tempo':
+                return args.tempo;
             case 'arbitrum':
                 return args.arbitrum;
             case 'stellar':
